### PR TITLE
For #26335: Enable sponsored shortcuts by default

### LINKED ISSUE #26335: Enable Sponsored Shortcuts by default
Jira ticket: https://mozilla-hub.atlassian.net/browse/FNXV2-20230

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21206)

### PR DESCRIPTION
**Pull request by Alexandru2909**
Enabled sponsored shortcuts by default for all users.

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #26335

### CODE REVIEW

File: nimbus.fml.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ features:
             "recently-saved": true,
             "recent-explorations": true,
             "pocket": true,
-            "contile-top-sites": false,
+            "contile-top-sites": true,
```
Comment (Mugurell) [inline nimbus.fml.yaml:22]: If the experiment ended could we just remove it from the app?
And then have the `showContileFeature` setting just become
```xml
    var showContileFeature by booleanPreference(
        key = appContext.getPreferenceKey(R.string.pref_key_enable_contile),
        default = true,
    )
```

Comment (Mugurell) [reply nimbus.fml.yaml:22]: Looks like this was the experiment for contile https://experimenter.services.mozilla.com/nimbus/firefox-android-sponsored-shortcuts-experiment with @lwa-moz being the owner.
Asking Loren if we need to keep supporting the experiment in the app or we can remove it with the experimentation about this feature done.

Comment (Mugurell) [reply nimbus.fml.yaml:22]: Got a confirmation on slack that it's safe to remove the experiment in 105 but this change should not be uplifted.

**Comment by Mugurell** [comment]
`ui-test-x86-debug` failed because of `verifyRunBlockingAndStrictModeSuppresionCount` which is already tracked in https://github.com/mozilla-mobile/fenix/issues/25142.
Restarting the task.

**Review by Mugurell — APPROVED** [review]

**Comment by Mugurell** [comment]
`ui-test-x86-debug` failed because of [java.lang.RuntimeException: Error while connecting UiAutomation](https://console.firebase.google.com/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6750958021445983470/executions/bs.e949fa1e5c522847/testcases/1), issue is already tracked in https://github.com/mozilla-mobile/fenix/issues/25925.
Restarting task.

**Comment by Mugurell** [comment]
`ui-test-x86-debug` failed again because of `verifyRunBlockingAndStrictModeSuppresionCount`.
Restarting the task.

**Comment by Mugurell** [comment]
`ui-test-x86-debug` failed again because of `verifyRunBlockingAndStrictModeSuppresionCount`.
Restarting the task. Again.

**Comment by gabrielluong** [comment]
Rerunning

**Comment by Mugurell** [comment]
@rvandermeulen This was discussed in todays planning to be a priority to land in 105.
Since the patch is simple and straighforward and there are separate some new intermittent failures affecting many new PRs - this included would you be able to help with manually merging this?

**Pull request merged**